.gitignore

```diff
@@ -29,6 +29,7 @@ vendor.bundle.js
 bundle.js.map
 dist/
 yarn-error.log
+/src/app/src/layer_data
```

Comment:
Consider naming this folder `data` and adding a `.gitkeep` file so the folder is present in the repo, even if the data is not.

Reply:
Good idea! What about using a `.gitignore` like this:
```.gitignore
# Ignore everything in this directory
*

# except this file
!.gitignore
```

Reply:
Ah. I like having a single root `.gitignore` that covers everything. Let's keep the entry there, and we can `git add --force src/app/src/data/.gitkeep` to add that one file while ignoring the rest.